Tribler/Test/Core/test_osutils.py

```diff
@@ -18,38 +18,38 @@ def test_fix_filebasename(self):
         default_name = '_'
         win_name_table = {
             'abcdef': 'abcdef',
-          '.': default_name,
```

Comment:
please, remove unrelated code reformatting from the PR. A mass reformatting will happen soon.